docker/pypi/wmagent-couchdb/README.md

```diff
@@ -0,0 +1,324 @@
+# Couchdb default image for running wmagent
+
+## Prerequisites
+
+This image inherits from the mainstream `couchdb` one, and follows the same
+tagging schema. On top of the base `couchdb` image we add all the structure
+needed for running the WMAgent with CouchDB and two main scripts:
+
+* `couchdb-docker-run.sh`
+* `couchdb-docker-build.sh`
+
+For building the containers, and for creating the mount area at the host and the
+the bind mounts inside the container, respectively. Those are as follows:
+
+* At the host:
+
+```
+/data/dockerMount/{admin|srv}/couchdb
+```
+
+* At the container:
+
+```
+/data/{admin|srv}/couchdb
+```
+
+Upon starting the container we try to initialize the default user and system
+databases, which if previously created should exist in the host mount area. And
+the last steps are creating the `wmagent` database.
```

Comment:
There is no such `wmagent` database. Did you mean that the databases required by WMAgent are created?